src/application/GlucoseGenie/GlucoseGenie/Recipes/View/RecipeUIView.swift

```diff
@@ -310,45 +322,31 @@ struct RecipeUIView: View {
             
             URLSession.shared.dataTask(with: request) { data, response, error in
                 if let error = error {
-                    print("Request Error: \(error.localizedDescription)")
-                    DispatchQueue.main.async {
-                        self.isLoading = false
-                        self.uiErrorMessage = "Network error."
-                    }
-                    return
-                }
-                
-                guard let httpResponse = response as? HTTPURLResponse else {
-                    print("No valid response")
                     DispatchQueue.main.async {
                         self.isLoading = false
-                        self.uiErrorMessage = "Something went wrong while getting recipes."
+                        self.uiErrorMessage = "Network error: \(error.localizedDescription)"
```

Comment:
I don't think we want the actual error to be written out to the user's display which is what `self.uiErrorMessage` does. It is a text display to the Recipe Screen of the user. We can print the error for logging but not to the display. 
```suggestion
                        self.uiErrorMessage = "Network error: Something went wrong while loading recipes."
```